client/src/client/spammer_client.py

```diff
@@ -0,0 +1,96 @@
+"""Spammer client service."""
+import asyncio
+import os
+import sys
+from random import choice
+
+from olm import Account
+
+from common.keys import generate_keys
+
+from .session_manager_client import SessionManager
+
+messages = [
+    "Beautiful is better than ugly.",
+    "Explicit is better than implicit.",
+    "Simple is better than complex.",
+    "Complex is better than complicated.",
+    "Flat is better than nested.",
+    "Sparse is better than dense.",
+    "Readability counts.",
+    "Special cases aren't special enough to break the rules.",
+    "Although practicality beats purity.",
+    "Errors should never pass silently.",
+    "Unless explicitly silenced.",
+    "In the face of ambiguity, refuse the temptation to guess.",
+    "There should be one-- and preferably only one --obvious way to do it.",
+    "Although that way may not be obvious at first unless you're Dutch.",
+    "Now is better than never.",
+    "Although never is often better than right now.",
+    "If the implementation is hard to explain, it's a bad idea.",
+    "If the implementation is easy to explain, it may be a good idea.",
+    "Namespaces are one honking great idea -- let's do more of those!",
+]
+
+
+class SpammerClient:
+    """Spammer client service."""
+
+    def __init__(self) -> None:
+        """Run a spammer client."""
+        self.server_hostname = os.environ["CANS_SERVER_HOSTNAME"]
+        self.server_port = os.environ["CANS_PORT"]
+        self.certpath = os.environ["CANS_SELF_SIGNED_CERT_PATH"]
+
+        self.event_loop = asyncio.get_event_loop()
+
+        account = Account()
+
+        private_key, public_key = generate_keys()
+        print(public_key)
```

Comment:
Would it not be better to print the hash of the key? This is what is ultimately used as the user identifier

Reply:
At the moment of writing this I thought public key would be enough, we can print all :)